lib/src/execute.rs

```diff
@@ -264,7 +264,10 @@ impl ExecuteCtx {
             .await
             .map(|_| ())
             .map_err(|trap| {
-                event!(Level::ERROR, "WebAssembly trapped: {}", trap);
+                // Be sure that we only log non-zero status codes.
+                if trap.i32_exit_status() != Some(0) {
+                    event!(Level::ERROR, "WebAssembly huhu: {}", trap);
```

Comment:
This LGTM except that I don't know what "huhu" is, was it intended?

Reply:
not intended, thank you for catching this! i've force pushed a fresh commit (_and commit message_) that doesn't include my silly testing string :sweat_smile: